src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpWhosInControlApplicationController.cs

```diff
@@ -219,6 +223,15 @@ public async Task<IActionResult> ConfirmTrustees(Guid applicationId)
         {
             var trusteesData = await _tabularDataRepository.GetTabularDataAnswer(applicationId, RoatpWorkflowQuestionTags.CharityCommissionTrustees);
 
+            var qnaApplicationData = await _qnaApiClient.GetApplicationData(applicationId);
+            var ukprn = qnaApplicationData.GetValue(RoatpWorkflowQuestionTags.UKPRN)?.Value<string>();
```

Comment:
Ukprn is already being passed in to this method, is there any other purpose for this call?

Reply:
I don't understand this comment, ukprn is not being passed into this method. only applicationId